artist-core/src/main/kotlin/com/uber/artist/KotlinArtistCodeGenerator.kt

```diff
@@ -74,7 +74,7 @@ class KotlinArtistCodeGenerator : ArtistCodeGenerator<FileSpec, TypeSpec.Builder
   override fun createInitBuilderFor(stencil: KotlinViewStencil, type: TypeSpec.Builder): FunSpec.Builder {
     return FunSpec.builder("init")
         .addAnnotation(KotlinTypeNames.Annotations.CallSuper)
-        .addModifiers(KModifier.OPEN, KModifier.PROTECTED)
+        .addModifiers(KModifier.PROTECTED, KModifier.OPEN)
```

Comment:
nice, thx for filing issue, too. how did you find this? ktlint, kotlin warning, or ide inspection?

Reply:
Just noticed eyeballing it. KotlinPoet is supposed to do this already though, so something else is going on